tools/intersmash-tools-provisioners/pom.xml

```diff
@@ -145,6 +145,16 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources/filtered</directory>
```

Comment:
@fabiobrz I don't see anything in `src/main/resources/filtered` ... just tools/intersmash-tools-provisioners/src/main/resources/org/jboss/intersmash/tools/provision/helm/values/schema/wildfly/helm-wildfly-release.json might be processed

Reply:
my bad, I didn't notice the location of that file